### PR TITLE
check max value allowed for setInterval in v8

### PR DESCRIPTION
**Pull request by AlvaroVega**
related https://github.com/telefonicaid/perseo-fe/issues/464

### CODE REVIEW

File: lib/models/noSignal.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ var util = require('util'),
     CHECK_INTERVAL = 8,
     context = { op: 'checkNonSignal', comp: constants.COMPONENT_NAME, corr: 'n/a', trans: 'n/a' },
     lastTime,
-    MIN_INTERVAL_MS = 30e3;
+    MIN_INTERVAL_MS = 30e3,
+    MAX_INTERVAL_MS = 2147483647;
```
Comment (fgalan) [inline lib/models/noSignal.js:49]: Maybe a little explanation about from where these values come as a code comment would be useful from the point of view of manteanibility.

Comment (AlvaroVega) [reply lib/models/noSignal.js]: fixed in https://github.com/telefonicaid/perseo-fe/pull/465/commits/a5401764000550724ff9a7de83be627c7d132e27

**Review by fgalan — APPROVED** [review]
LGTM (with a small addition to documentation from my own)

**Comment by fgalan** [comment]
Travis is failing...

**Comment by AlvaroVega** [comment]
....just a glitch

**Pull request merged**